docs/Deprecated-functionality.rst

```diff
@@ -5,9 +5,39 @@ Deprecated functionality
 
 Some of the functionality that was available in previous EasyBuild versions is *deprecated*, and should no longer be used.
 
-This page describes the :ref:`deprecation policy <deprecation_policy>` in EasyBuild, explains how to easily :ref:`check
-whether you are still relying on deprecated functionality <how_to_check_use_of_deprecated_functionality>`, and provides
-an :ref:`overview of currently deprecated functionality <overview_deprecated>`, together with available alternatives.
+Support for using this functionality will be dropped in a future version of EasyBuild (see :ref:`no_longer_supported`
```

Comment:
remove: "support for using"

consider using "removed" instead of "dropped"

leaving: "This functionality will be dropped in a future version of EasyBuild"